DOCSP-23699: fixed format of a heading (#374)
mongodb/docs-node: source/fundamentals/crud/read-operations/distinct.txt

Distinct
--------

The ``distinct()`` method requires one document field parameter. You can specify two optional 
parameters to adjust the method output:

- A ``query`` parameter to refine your results 
- An ``options`` parameter to set collation rules

Document Field Parameter
~~~~~~~~~~~~~~~~~~~~~~~~

> Use a document field name as a parameter for the ``distinct()`` method to return a 
> list of the field's unique values.

Reviewer: Suggestion:
Here's a more concise way to word the sentence.

```suggestion
Pass the name of the document field to return a list of the field's unique values.
```
